server/index.js

```diff
@@ -19,6 +19,12 @@ app.get('/api/health-check', (req, res, next) => {
     .catch(err => next(err));
 });
 
+app.get('/api/products', (req, res, next) => {
+  db.query('select "productId", "name", "price", "image", "shortDescription" from "products"')
```

Comment:
Great work!! You can also save your query to a const inside of your app.get and then pass that const to the db.query.